utils/usage/usage.go

```diff
@@ -89,6 +93,14 @@ func (ur *UsageReporter) Report(features ...ReportFeature) {
 			return
 		})
 	}
+	if ur.sendToXray {
+		ur.reportWaitGroup.Go(func() (err error) {
+			if err = ur.reportToXray(features...); err != nil {
+				err = fmt.Errorf("xray, %s", err.Error())
```

Comment:
Let's do it in "ecosystem" and "artifactory" too
```suggestion
				err = fmt.Errorf("xray, %w", err)
```